modules/nf-core/mygene/templates/mygene.py

```diff
@@ -158,9 +197,9 @@ def query2idmap(self) -> dict:
             species=self.species,
             entrezonly=self.entrezonly,
             ensemblonly=self.ensemblonly,
-            returnall=True
+            returnall=True,
```

Comment:
Why does this comma get added here?

Reply:
Optional commas after the last expression are a fantastic invention:
 * you can comment/uncomment any line and it will still work
 * you can't forget to add a comma when adding an entry.

The goal of black/ruff is also to minimize diffs. With the extra comma, when an item gets added, only one line has changed, while it would be two if you'd need to add the comma first.